src/Engine/ProtoCore/SyntaxAnalysis/AstVisitor.cs

```diff
@@ -239,7 +266,7 @@ public virtual TResult VisitGroupExpressionNode(GroupExpressionNode node)
 
```

Comment:
This is as expected. `AssociaitveAstVisitor` is for `AssociativeNode`, right now for imperative node it doesn't have `Accept()` method.